src/section-multiplying-polynomials.ptx

```diff
@@ -51,14 +37,7 @@
 
     <subsection>
         <title>Multiplying Monomials</title>
-            <p>We'll start our study of polynomial multiplication by multiplying monomials (one-term polynomials).  Once we understand how monomial multiplication works, we'll have half of the foundation for general polynomial multiplication.</p>
-            <p>When we multiply the monomials <m>2</m> and <m>xy</m>, which approach is correct?<md>
-                <mrow>\amp\text{Approach 1: }2(xy)=2xy</mrow>
-                <mrow>\amp\text{Approach 2: }2(xy)=2x\cdot2y=4xy</mrow>
-            </md>The questions really at hand are: Do we distribute the <m>2</m> to both the <m>x</m> and <m>y</m>?  Is this a situation where distribution is appropriate?</p>
-            <p>If we quickly review the distributive property !!!(link to distribution section?)!!!, we'll remember that distribution only applies when we have an expression with multiplication outside a set of parentheses and addition (or subtraction) inside the parentheses.</p>
-            <p>Since our expression <m>2(xy)</m> does not involve addition or subtraction anywhere, distribution would not be appropriate to use in this situation.  Therefore Approach 1 is correct.<me>2(xy)=2xy</me></p>
-            <p>Since monomials do not involve any operation other than multiplication, multiplying monomials will only make use of the commutative and associative properties.  In general, you will not show where you've used these two properties.  That work is primarily done mentally.</p>
+            <p>When we multiply the monomials <m>2</m> and <m>xy</m>, note that we have only a single term being multiplied by a single term. Simply multiply the coefficients with the coefficients and combine variables using the product rule !!!insert ref to product rule!!!. Thus we have that <m>2(xy)=2xy</m>. Please remember that you only use the distributive property !!!(link to distribution section)!!! across addition or subtraction, never multiplication or division. This is why the <m>2</m> doesn't <q>distribute</q> to both the <m>x</m> and <m>y</m> separately.</p>
```

Comment:
The triple exclamation point notes here could get forgotten about and not seen by us before a student sees them. Do like this instead: `<!-- TODO: insert ref to product rule-->`. We won't miss these because we will scan for comments with "TODO".